### DISCUSSION TITLE
Terraform git-tags skipReason depIndex

### DISCUSSION THREAD

**Discussion by renan-alm** [renovatebot/renovate#12430] [Request Help] [answered]
Hi,

I'm currently using a self-hosted instance of Renovate to monitor Terraform and I am receiving the below error: 

               {
                 "depType": "module",
                 "depName": "<Bitbucket_host>/bucket-tf-module",
                 "lookupName": "ssh://git@<Bitbucket_host>/bucket-tf-module.git",
                 "currentValue": "v19",
                 "datasource": "git-tags",
                 "depIndex": 2,
                 "updates": [],
                 "warnings": [],
                 "versioning": "hashicorp",
                 "currentVersion": null
               },
               {"skipReason": "invalid-value", "depIndex": 3, "updates": []}

My config looks like this:

        {
            "matchDepTypes": ["git-tags"],
            "groupName": "terraform-git-tags",
            "commitMessagePrefix": "[skip app]",
            "commitMessageTopic": "terraform git-tags",
            "lockFileMaintenance": {
                "enabled": true
            }
        }

What do I need to adjust on my config in order to change the depIndex?

**Comment by rarkins** [comment, marked as answer]
Depindex is not the error. v19 is the invalid value. What type of versioning scheme is in use for this dependency? Ie it's not semver

**Reply by renan-alm** [reply]
Using Git. That's why I used "matchDepTypes" set to "git-tags" in this context.

Would that be the correct mindset?

**Reply by rarkins** [reply]
Git is not a versioning scheme. Examples of versioning schemes: semver, calver, maven, pep440, etc.

You can perhaps try switching this dependency to use Renovate's `loose` versioning

**Reply by renan-alm** [reply]
I will change my semver to Maven to make it easier for Renovate.
Thank you for your input!
